content/chainguard/administration/assumable-ids/identity-examples/github-identity/index.md

```diff
@@ -10,7 +10,7 @@ lead: ""
 description: "Procedural tutorial outlining how to create a Chainguard Enforce identity that can be assumed by a GitHub Actions workflow."
```

Comment:
We took out "procedural" below, so I'd take it out here as well. Also remove "enforce"